LongestPole/Main.lean

```diff
@@ -31,51 +31,51 @@ def runCmd (cmd : String) (args : Array String) (throwFailure := true) : IO Stri
 def runCurl (args : Array String) (throwFailure := true) : IO String := do
   runCmd "curl" args throwFailure
 
-def speedCenterRunJson (hash : String) : IO String :=
+def SpeedCenterAPI.RunJson (hash : String) : IO String :=
   -- e7b27246-a3e6-496a-b552-ff4b45c7236e is the `repo_id` for the `mathlib4` repository.
   runCurl #["http://speed.lean-fro.org/mathlib4/api/run/e7b27246-a3e6-496a-b552-ff4b45c7236e?hash=" ++ hash]
 
-structure SpeedCenterSource_1 where
+structure SpeedCenterAPI.Source_1 where
```

Comment:
```suggestion
structure SpeedCenterAPI.Source1 where
```
to respect the naming convention

Reply:
https://github.com/leanprover-community/mathlib4/pull/8361#discussion_r1573881736